src/lib.rs

```diff
@@ -812,6 +894,7 @@ impl Drop for Savefile {
 }
 
 #[cfg(not(windows))]
+/// Open a raw file descriptor.
```

Comment:
Is this function intended to be public API? It looks like `Capture<Offline>::from_raw_fd` and `Capture<Activated>::savefile_raw_fd` would be the preferred public API.